tmt/trying.py

```diff
@@ -527,6 +532,35 @@ def action_quit(self, plan: Plan) -> None:
         run_id = style(str(plan.my_run.workdir), fg="magenta")
         self.print(f"Run {run_id} successfully finished. Bye for now!")
 
+    def action_host(self, plan: Plan) -> None:
+        """
+        Run command on the host
+        """
+
+        while True:
+            self.print(style(f"Enter command (or '{Action.QUIT.key}' to quit): ", fg="green"))
+            try:
+                raw_command = input("> ")
+            except (KeyboardInterrupt, EOFError):
+                self.print("Exiting host command mode. Bye for now!")
+                break
+
+            if not raw_command or raw_command == Action.QUIT.key:
+                self.print("Exiting host command mode. Bye for now!")
+                break
+
+            # Execute the command on the host
+            try:
+                command = Command(*shlex.split(raw_command))
+                command.run(cwd=plan.workdir, logger=self._logger, interactive=True)
+            except tmt.utils.RunError as error:
+                tmt.utils.show_exception_as_warning(
+                    exception=error,
+                    message=error.message,
```

Comment:
Exception message would be logged as part of the traceback by default. We can use this parameter to provide more contextual message than possibly a raw exception output. E.g. "Failed to run command on the guest" or something similar.